lib/room.cpp

```diff
@@ -430,11 +432,22 @@ QString Room::name() const
     return d->getCurrentState<RoomNameEvent>()->name();
 }
 
+QStringList Room::aliases() const
+{
+    QStringList result(d->getCurrentState<RoomCanonicalAliasEvent>()->alias());
+    result += d->getCurrentState<RoomCanonicalAliasEvent>()->altAliases();
+    return result;
```

Comment:
```suggestion
    const auto* evt = d->getCurrentState<RoomCanonicalAliasEvent>();
    return QStringList(evt->altAliases()) << evt->alias();
```
That puts the canonical alias at the end (vs. at the start) of the string list but avoids one-by-one copying. And extracting a variable helps making the code less verbose.